cmd/logs.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"github.com/astronomer/astro-cli/logs"
+	"github.com/spf13/cobra"
+)
+
+// logsCmd represents the logs command
+var (
+	search string
+	follow bool
+
+	logsCmd = &cobra.Command{
+		Use:     "logs",
+		Aliases: []string{"log", "l"},
+		Short:   "show logs",
+		Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+astro logs webserver`,
+	}
+
+	webserverLogsCmd = &cobra.Command{
+		Use:     "webserver",
+		Aliases: []string{"web", "w"},
+		Short:   "airflow webserver logs",
+		Long:    "airflow webserver logs",
+		Args:    cobra.ExactArgs(1),
+		RunE:    webserverRemoteLogs,
+	}
+
+	schedulerLogsCmd = &cobra.Command{
+		Use:     "scheduler",
+		Aliases: []string{"sch", "s"},
+		Short:   "airflow scheduler logs",
+		Long:    "airflow scheduler logs",
+		Args:    cobra.ExactArgs(1),
+		RunE:    schedulerRemoteLogs,
+	}
+
+	workersLogsCmd = &cobra.Command{
+		Use:     "workers",
+		Aliases: []string{"workers", "worker", "wrk"},
+		Short:   "airflow workers logs",
+		Long:    "airflow workers logs",
+		Args:    cobra.ExactArgs(1),
+		RunE:    workersRemoteLogs,
+	}
+)
+
+func init() {
+	RootCmd.AddCommand(logsCmd)
+	webserverLogsCmd.Flags().StringVarP(&search, "search", "s", "", "Search term inside logs")
```

Comment:
Small, but one flag ends with a period and one doesn't. I don't have much preference, other than we match everything else :)